docs/customizations/helm.rst

```diff
@@ -37,102 +37,155 @@ General Parameters
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].preference.matchExpressions[0].operator
+     - string
+     - `"In"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].preference.matchExpressions[0].values[0]
+     - string
+     - `""`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[0].weight
+     - int
+     - `1`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].preference.matchExpressions[0].key
+     - string
+     - `"node-role.kubernetes.io/control-plane"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].preference.matchExpressions[0].operator
+     - string
+     - `"In"`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].preference.matchExpressions[0].values[0]
+     - string
+     - `""`
+     -
+   * - operator.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution[1].weight
+     - int
+     - `1`
+     -
+   * - operator.cniBinDirectory
+     - string
+     - `"/opt/cni/bin"`
+     -
+   * - operator.fullnameOverride
+     - string
+     - `""`
+     -
+   * - operator.image
+     - string
+     - `"network-operator"`
+     -
+   * - operator.nameOverride
+     - string
+     - `""`
+     -
+   * - operator.nodeSelector
+     - object
+     - `{}`
+     -
+   * - operator.repository
+     - string
+     - `"nvcr.io/nvidia/cloud-native"`
+     -
+   * - operator.resources.limits.cpu
+     - string
+     - `"500m"`
+     -
+   * - operator.resources.limits.memory
+     - string
+     - `"128Mi"`
+     -
+   * - operator.resources.requests.cpu
+     - string
+     - `"5m"`
+     -
+   * - operator.resources.requests.memory
+     - string
+     - `"64Mi"`
+     -
+   * - operator.tolerations[0].effect
+     - string
+     - `"NoSchedule"`
+     -
+   * - operator.tolerations[0].key
+     - string
+     - `"node-role.kubernetes.io/master"`
+     -
+   * - operator.tolerations[0].operator
+     - string
+     - `"Equal"`
+     -
+   * - operator.tolerations[0].value
+     - string
+     - `""`
+     -
+   * - operator.tolerations[1].effect
+     - string
+     - `"NoSchedule"`
+     -
+   * - operator.tolerations[1].key
+     - string
+     - `"node-role.kubernetes.io/control-plane"`
+     -
+   * - operator.tolerations[1].operator
+     - string
+     - `"Equal"`
+     -
+   * - operator.tolerations[1].value
+     - string
+     - `""`
+     -
+   * - operator.useDTK
+     - bool
+     - `true`
+     -
    * - sriovNetworkOperator.enabled
-     - Bool
-     - False
-     - Deploy SR-IOV Network Operator
-   * - sriovNetworkOperator.configDaemonNodeSelectorExtra
-     - List
-     - node-role.kubernetes.io/worker: ""
-     - Additional values for SR-IOV Config Daemon nodes selector
+     - bool
+     - `false`
+     -
+   * - test.pf
```

Comment:
Is it supposed to be skipped?

Reply:
It will be skipped, once we merge [1011](https://github.com/Mellanox/network-operator/pull/1011)